presentation/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -0,0 +1,113 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import classnames from "classnames";
+import Component, {
+  components, ControlProps, IndicatorProps, MenuProps, MultiValueProps, OptionProps, Props, ValueContainerProps,
+} from "react-select";
```

Comment:
Is there an equivalent component we could use from iTwinUI-react? If not, we should file an issue for one.

Reply:
I could not find a component that I could use from iTwinUI-react.[ There is issue](https://github.com/iTwin/iTwinUI-react/issues/344) filled already and added this component to have it temporarily.